src/require.ts

```diff
@@ -31,7 +31,7 @@ export function jitiRequire(
   }
 
   // Check for builtin node module like fs
-  if (builtinModules.includes(id) || id === ".pnp.js" /* #24 */) {
+  if (isBuiltin(id) || id === ".pnp.js" /* #24 */) {
```

Comment:
The [`isBuiltIn`](https://nodejs.org/api/module.html#moduleisbuiltinmodulename) utility is available since Node.js >=16.17.0. We might need to be careful about backward compatibility and i guess with your fix above it might not be necessary anymore.

Reply:
`isBuiltin()` is [already used](https://github.com/search?q=repo%3Aunjs%2Fjiti%20isBuiltin&type=code) in the codebase.